Core/clim-core/gadgets.lisp

```diff
@@ -1164,11 +1164,10 @@ and must never be nil.")
 
 (defmethod handle-event ((pane push-button-pane) (event pointer-button-release-event))
   (with-slots (armed pressedp) pane
+    (when (and armed pressedp)
+      (activate-callback pane (gadget-client pane) (gadget-id pane)))
     (setf pressedp nil)
-    (when armed
-      (activate-callback pane (gadget-client pane) (gadget-id pane))
-      (setf pressedp nil)
-      (dispatch-repaint pane +everywhere+))))
+      (dispatch-repaint pane +everywhere+)))
```

Comment:
Indentation looks funny here. Should all three forms be inside the `when`?

Reply:
done (they should not)